app/src/ui/changes/no-changes.tsx

```diff
@@ -697,8 +697,8 @@ export class NoChanges extends React.Component<
             <div className="text">
               <h1>No local changes</h1>
               <p>
-                You have no uncommitted changes in your repository! Here are
-                some friendly suggestions for what to do next.
+                There are no uncommitted changes to this repository. 
+                Here are some friendly suggestions on what to do next:
```

Comment:
The second part of this is also a bit about stripping out superfluous words, and trying a different phrasing:

Yes, I'm being ironic by using "superfluous". It's fine.

```suggestion
                Here are some actions you may find useful:
```

Thoughts?

Reply:
Sounds much nicer!